packages/auth/src/ActivateAccount/ActivateAccountForm.tsx

```diff
@@ -34,15 +35,23 @@ export const ActivateAccountForm: FC<ActivateAccountFormProps> = (props) => {
   const {
     activateState: { loading, error },
     activateAccount,
+    forgotPasswordState: { passwordConfig },
+    loadPasswordConfig,
+    resetForgotPasswordState,
   } = authState;
   if (renderer) {
     return createElement(renderer, { ...props, ...authState });
   }
 
+  useEffect((): (() => void) => {
+    loadPasswordConfig();
+    return resetForgotPasswordState;
```

Comment:
What is this?

Reply:
we need to load the password configuration for activateAccount as well. so I'm loading the password config here as well